### PR TITLE
br/lighting: add multi-valued index test

### PR DESCRIPTION
**Pull request by xiongjiwei**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/42037

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/tests/lightning_multi_valued_index/run.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+UNDO_CMD=$(read_result)
+undo_set_globals() {
+    run_sql "$UNDO_CMD"
+}
+trap undo_set_globals EXIT
+
+# There is normally a 2 second delay between these SET GLOBAL statements returns
+# and the changes are actually effective. So we have this check-and-retry loop
+# below to ensure Lightning gets our desired global vars.
+run_sql "SET GLOBAL time_zone='-08:00', GLOBAL default_week_format=4, GLOBAL block_encryption_mode='aes-256-cbc'"
```
Comment (lance6716) [inline br/tests/lightning_multi_valued_index/run.sh:39]: why we need to test this?

**Review by lance6716 — COMMENTED** [review]
rest lgtm for lightning part

File: br/tests/lightning_multi_valued_index/run.sh (hunk 1 above)
Comment (xiongjiwei) [reply br/tests/lightning_multi_valued_index/run.sh:39]: just copy from other files

Comment (lance6716) [reply br/tests/lightning_multi_valued_index/run.sh:39]: I think it's OK to keep or remove them

**Review by lance6716 — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**